inc/spbc-settings.php

```diff
@@ -5566,6 +5710,26 @@ function spbc__get_accordion_tab_info_block_html($for)
                 ';
             $button_div .= '</div>';
             break;
+        case 'critical-for-widget':
+            //banner for widget if critical files exist
+            $info_block_out = __('With a high degree of probability, your site has been infected. If you need professional help 
+        from security specialists, feel free to order', 'security-malware-firewall');
+
+            $classes = 'spbc_widget_notice_critical';
+            $show_exclaim_triangle = true;
+
+            //generate button
+            $landing_page_link = 'https://l.cleantalk.org/website-malware-removal?email=' . esc_attr($email) . '&website=' . esc_attr($website);
+            $button_text = __('Request Malware removal', 'security-malware-firewall');
+            $button_div = '<div style="">';
+            $button_div .= '
+                <a class="spbc_manual_link" target="_blank" href="' . $landing_page_link . '">'
+                . '<i class="spbc-icon-link-ext"></i>&nbsp;&nbsp;'
+                . $button_text
+                . '</a>
+                ';
+            $button_div .= '</div>';
```

Comment:
Is it possible to use the new [`LinkConstructor`](https://github.com/CleanTalk/security-malware-firewall/pull/389/files#diff-22b5810a774313389a32bb2f031865f7c0a2de007eb6ea04e1d0cfa4f7aec812) class?